resources/builder.inc.sh

```diff
@@ -0,0 +1,690 @@
+#!/usr/bin/env bash
+#
+# This script contains utilities for builder_script calls
+#
+# * builder_ functions and variables are defined here.
+# * REPO_ROOT defines the top level of this repository
+# * THIS_SCRIPT_PATH defines the full path of the running script
+# * THIS_SCRIPT_NAME defines the basename of the running script
+# * THIS_SCRIPT_IDENTIFIER defines the repo-relative path of the running script
+# * _builder_ functions and variables are internal use only for this script 
```

Comment:
I think it's okay as it is? `_` is a common internal prefix for many things.

Reply:
I initially found it a bit confusing what's meant with `this script` since we have two levels of included scripts. `builder.inc.sh` defines functionality that `build-utils.sh` uses. 
"Internal" could mean that it's only to be used by `build-utils.sh`, but is not part of the API, or it could mean that it's only to be used by `builder.inc.sh`. It became clear after I read through the whole of `builder.inc.sh` but at first I wasn't sure.

Reply:
Gotcha. I think we update the wording in this comment to clarify the include chain that is currently common in the keyman repo:

```suggestion
# * _builder_ functions and variables are internal use only for builder.inc.sh, and 
# subject to change at any time. Do not use them in other scripts.
# * Note: the running script is the top-level script that includes either 
#   builder.inc.sh directly, or, just in the Keyman repo, via build-utils.sh.
```